app/src/klarNightly/res/xml-v25/shortcuts.xml

```diff
@@ -11,7 +11,7 @@
         android:shortcutLongLabel="@string/shortcut_erase_long_label">
         <intent
             android:action="erase"
-            android:targetPackage="${applicationId}"
+            android:targetPackage="org.mozilla.klar"
```

Comment:
Should we be hardcoding the applicationId? It's already coming from the build configuration.

Reply:
I hardcoded it to keep it consistent with the convention followed in the other shortcuts.xml files for the various build variants